p5_build

```diff
@@ -25,11 +25,11 @@ if [ -d "package/translate" ]; then
 	echo "[build] translate dir found, running merge."
 	(cd package/translate && sh ./merge)
 	(cd package/translate && sh ./build)
-	if [ "$(git diff --stat .)" != "" ]; then
-		echo "[build] Changed detected. Cancelling build."
-		git diff --stat .
-		exit 1
-	fi
+	#if [ "$(git diff --stat .)" != "" ]; then
+	#	echo "[build] Changed detected. Cancelling build."
+	#	git diff --stat .
+	#	exit 1
+	#fi
```

Comment:
I'll be needing to change to a `plasma5` branch just to create a plasma5 widget, so we can just remove/edit the old scripts in the `master`/`plasma6` branch.